controller/manager/manager.go

```diff
@@ -244,29 +244,27 @@ func (m DefaultManager) ScaleContainer(id string, numInstances int) ScaleResult
 		return result
 	}
 
-	for i := 1; i < numInstances; i++ {
+	for i := 0; i < numInstances; i++ {
 		go func(instance int) {
-			for {
```

Comment:
this infinite loop was causing more than 1 container to be created per iteration